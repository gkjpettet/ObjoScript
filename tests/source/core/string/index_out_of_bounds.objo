System.print("hello"[5]) # Expect error