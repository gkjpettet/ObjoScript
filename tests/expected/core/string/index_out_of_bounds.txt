The index is out of bounds (5).